control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -634,6 +639,13 @@ func useHCPRouter(hostedControlPlane *hyperv1.HostedControlPlane) bool {
 }
 
 func (r *HostedControlPlaneReconciler) reconcile(ctx context.Context, hostedControlPlane *hyperv1.HostedControlPlane, createOrUpdate upsert.CreateOrUpdateFN, releaseImage *releaseinfo.ReleaseImage, infraStatus InfrastructureStatus) error {
+	if useHCPRouter(hostedControlPlane) {
```

Comment:
Commit says >Make router service reconciliation part of infra reconciliation.

Should this belong to `reconcileInfrastructure`?

Reply:
Ah, I see you broke it down via `reconcileHCPRouterServices`